Use_Cases/Opportunity_List/README.md

```diff
@@ -0,0 +1,28 @@
+# Opportunity List
+
+Easily view opportunities in a single screen both in chart and table format
+
+<img  src="opportunity_list.png"  width="300"></img>
+
+The table shows all opportunities while the charts and KPIs offer an overview. Search or filter to drill down to the opportunities you want to see, or click on portions of a chart to drill down and see the opportunities in that section. In addition, create a new opportunity using a step-by-step process.
+
+## Instructions
+- Page API: V2
+
+- Data source: Uses default Salesforce data source
+
+- Design system: [Download this Design System file](CompactOpportunityMgt.designsystem). Use the Import function on the Design System page to add this design system to your org.
+
+- Page XML: [Copy the XML from this page](Opportunity_List.xml), or save it as an XML file, and upload it as a new page in your Salesforce Org.
+
+## Notes
+- The chart allows you to drill down into the underlying Opportunity data and make changes if necessary.
```

Comment:
Do we need this? Don't we say it in the intro?

Reply:
Yep, that's a good point, I'll take it out